packages/cadl-rlc-emitter/test/unit/generate/docs.spec.ts

```diff
@@ -162,4 +166,39 @@ describe("Doc generation testing", () => {
       );
     });
   });
+
+  describe("docs in clientDefinitions.ts", () => {
+    it("should generate operation description", async () => {
+      const clientDef = await emitClientDefinitionFromCadl(
+        `
+        @summary("This is a summary")
+        @doc("This is the longer description")
+        op read(): {};
+        `
+      );
+      assert.ok(clientDef);
+      assertEqualContent(
+        clientDef?.content!,
+        `
+        import { ReadParameters } from "./parameters";
+        import { Read204Response } from "./responses";
+        import { Client, StreamableMethod } from "@azure-rest/core-client";
+  
+        export interface Read {
+            /** This is the longer description */
+            get(options?: ReadParameters): StreamableMethod<Read204Response>;
+        }
+  
+        export interface Routes {
+            /** Resource for '/' has methods for the following verbs: get */
+            (path: "/"): Read;
+        }
+        
+        export type testClient = Client & {
+                path: Routes;
```

Comment:
Can we use prettier and compare with the result after prettier ?

Reply:
updated